presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -348,6 +359,15 @@ public boolean supportClientTransaction()
         return queryRequestMetadata;
     }
 
+    /**
+     * Extracts a header from a headers map in the for of List<String>. Breaks each value with a comma, removes spaces and omits empty string.
```

Comment:
Where is the summary? We should have a summary before complete descriptions.
Also your line seems too long, break it please